src/VolleyManagement.WebClient/spa/app/app.component.html

```diff
@@ -1,3 +1,8 @@
-Volley-Management App
-
-<schedule [scheduleId]="13"></schedule>
\ No newline at end of file
+<div class="body">
+  <h1 class="appTitle">Volley-Management App</h1>
+  <div class="contentBody">
+    <vm-menu></vm-menu>
+    <router-outlet></router-outlet>
+    <!-- <schedule [scheduleId]="1"></schedule> -->
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)